lang/rs/examples/fwd/src/main.rs

```diff
@@ -278,7 +278,21 @@ fn tx_only(port: &Port, pkts: &mut [Packet]) -> Result<(), CneError> {
     let alloc_pkts = port.prepare_tx_packets(pkts)? as usize;
 
     // Fill a byte array with some values.
-    let input = "fd3c78299efefd3c00450008b82c9efe110400004f122e00a8c00100a8c01e221a002e16d2040101706f6e6d6c6b9a9e787776757473727131307a79";
+    // IPv4/UDP 64 byte packet
+    // TTL/Port Src/Dest   :       64/ 1234/ 5678
```

Comment:
same comment as in main.c, this is just a regular packet, but you're talking about VLAN and VxLAN in the comment which is a little confusing.